### PR TITLE
Score websocket

### PR DESCRIPTION
**Pull request by rteusner**

### CODE REVIEW

File: app/assets/javascripts/editor.js.erb [hunk 1]
```diff
@@ -184,8 +185,8 @@ $(function() {
     (streamed ? evaluateCodeWithStreamedResponse : evaluateCodeWithoutStreamedResponse)(url, callback);
   };
```
Comment (tstaubitz) [inline app/assets/javascripts/editor.js.erb]: Please remove server side event logic completely.

File: lib/docker_client.rb [hunk 2]
```diff
@@ -200,7 +200,7 @@ def execute_arbitrary_command(command, &block)
     execute_command(command, nil, block)
   end
 
-  #only used by server sent events (deprecated?)
+  #only used by score
   def execute_command(command, before_execution_block, output_consuming_block)
```
Comment (tstaubitz) [inline lib/docker_client.rb]: Have a look whether score calls execute_command

**Comment by tstaubitz** [comment]
apart from that 👍

File: lib/docker_client.rb (hunk 2 above)
Comment (rteusner) [inline lib/docker_client.rb]: I had a look.

**Pull request merged**